src/fixate/drivers/dmm/keithley_6500.py

```diff
@@ -40,7 +40,16 @@ def __init__(self, instrument, *args, **kwargs):
             "continuity": "CONT",
             "diode": "DIOD",
         }
-
+        self._nplc_modes = [
+            "voltage_dc",
+            "current_dc",
+            "resistance",
+            "fresistance",
+            "diode",
+            "temperature",
+        ]
```

Comment:
The Keithley driver does not have ``self._nplc_settings``. Does this mean I can pick any float between 0.0005 and 12?

I can see this becoming an issue when someone builds a test script with the Keithley and sets nplc = 3 for example. This will cause the Fluke to error because 3 is not an available setting. 
This is very much related to the range settings problem I had a think about earlier in the year. https://github.com/PyFixate/Fixate/pull/205

Reply:
That is correct. This is why I have error checking in the fluke DMM driver set_NPLC function to check that it is a specific setting where as in the keithley it just checks that it is within the instruments range. 

This will mean that if you try to set NPLC to 3, the keithley driver will run fine, but the fluke driver will throw a parameter error (and not try and set NPLC to an invalid setting)

Reply:
Unless there is a compelling reason driven by test requirements, is there a reasonable lowest common denominator we support here, even if that means the full range of values isn't available

Reply:
Ideally, a script wouldn't need to be tweaked if we swap between Fluke and Keithly. So, perhaps we limit to the intersection of the capabilities of each instrument?
.e.g. `[0.02, 0.2, 1, 10]`

Reply:
> Ideally, a script wouldn't need to be tweaked if we swap between Fluke and Keithly. So, perhaps we limit to the intersection of the capabilities of each instrument? .e.g. `[0.02, 0.2, 1, 10]`

This has been done as of 4d00493